server/src/service/InventoryService.ts

```diff
@@ -0,0 +1,35 @@
+import { InventoryItem } from "../models/store/inventoryItem";
+import { IInventoryRepo } from "../repositories/Store/inventoryRepository";
+
+export class InventoryService {
+    inventoryRepo: IInventoryRepo;
+   
+    constructor(repo: IInventoryRepo) {
+      this.inventoryRepo = repo;
+    }
+   
```

Comment:
These service classes seem a little pointless atm, but the idea is that if we need to multiple calls on a repo, any sort of logic, or error handling, I would rather have that code in here than cluttering the resolver

Reply:
That was my initial reaction, but I agree we're definitely going to need business logic and error handling in the near future. I'm becoming more on board with the whole service & repos pattern as I understand it more.